lib/rspec/rails/matchers/send_email.rb

```diff
@@ -0,0 +1,100 @@
+module RSpec
+  module Rails
+    module Matchers
+      # @api private
+      #
+      # Matcher class for `send_email`. Should not be instantiated directly.
+      #
+      # @see RSpec::Rails::Matchers#send_email
+      class SendEmail < RSpec::Rails::Matchers::BaseMatcher
+        def initialize(criteria)
+          normalized = %i[to from].map { |attr| criteria.key?(attr) ? [attr, Array(criteria[attr])] : nil }.compact.to_h
+          @criteria = { **criteria, **normalized }
```

Comment:
I think this would be more readable if you did this check at match time e.g. (and see: )
```suggestion
          @criteria = criteria
```